src/psyclone/psyGen.py

```diff
@@ -4928,6 +4928,19 @@ def _if_stmt_handler(self, node, parent):
                            nodes_parent=node)
         return ifblock
 
+    def _return_handler(self, node, parent):
+        '''
+        Transforms an fparser2 Return_Stmt to the PSyIRe representation.
+
+        :param child: node in fparser2 AST.
```

Comment:
Should 'child' here (and on the line below) be 'node'?

Reply:
Ups, that is wrong in all handlers. Fixed all.